frontend/app/tests/e2e/pages/user-security-settings-page.ts

```diff
@@ -27,7 +27,7 @@ export class UserSecuritySettingsPage {
   }
 
   confirmSuccess() {
-    cy.get('.message-dialog__title').should('include.text', 'Success');
+    cy.get('.message-dialog__title').should('include.text', 'ASDDASD');
```

Comment:
ASDDASD? :rofl:

Reply:
well, it will fail the test, won't it? :P